applications/nrf_desktop/configuration/nrf5340dk_nrf5340_cpuapp/click_detector_def.h

```diff
@@ -16,8 +16,10 @@
 const struct {} click_detector_def_include_once;
 
 static const struct click_detector_config click_detector_config[] = {
+#ifdef CONFIG_BT
```

Comment:
We need to depend on BLE peer control here (we can enable Bluetooth but keep the BLE peer control disabled). We also should update all of the click_detector configurations in similar way.

Discussed with @nordicjm. I will provide a complete fix.

Reply:
@MarekPieta need review on this PR